### PR TITLE
data: ta-sakin

### PR DESCRIPTION
**Pull request by ta-sakin**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4705"></a>

### CODE REVIEW

File: data/ta-sakin.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Tahjib Al Sakin",
+  "type": "personal",
+  "bio": "Software Developer | MERN | Graphql",
+  "avatar": "https://github.com/ta-sakin.png"
```
Comment (swagataroy30) [inline data/ta-sakin.json:5]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**